examples/README.md

```diff
@@ -42,18 +43,21 @@ modelname
 - `modelname_config.py`: This file should contain most of the configuration for
   the model architecture, learning rate, etc, using simple Python constants. We
   would like to avoid complex configuration setups (json, yaml, etc).
-- `modelname_preprocess.py`: If necessary. Standalone script to preprocess
-  inputs. If possible, prefer doing preprocessing dynamically with tf.data
-  inside the training and finetuning scripts.
 - `modelname_model.py`: This file should contain the actual `keras.Model` and
   any custom layers needed for the example. Use KerasNLP components where ever
   possible.
+- `modelname_preprocess.py`: If necessary. Standalone script to preprocess
```

Comment:
I might miss something - what is "If necessary"?

Reply:
"If necessary" = "we prefer not to have this script." Ideally preprocessing should live inside of the `train.py` and `finetune_X.py` and should not be a standalone invokable script.

We are forced to have this for BERT because NSP preprocessing cannot be done easily as a dynamic transformation of a dataset iterator.

Note that in this PR I was just changing the order of this bullet point to match the structure show above. There's not a word changed. But happy to update our language if we have a different way to say this!